### ISSUE TITLE
Modify connect function to receive headers parameter

### ISSUE THREAD

**Issue by mdelclaro** [heminei/php-websocket#8]
Hello, I would like to pass some extra header to the socket connection, like "Sec-websocket-protocol": "access_token<token>" for example. I've tried to extend the class and override the method but no success. 

Thank you very much.

**Comment by mdelclaro** [comment]
Something like:

```php
$headers[] = "Sec-websocket-protocol: access_token, " . $token;

public function connect(string $path = "/", string $origin = null, $headers)
    {
     ...
        $header .= "Sec-WebSocket-Key: " . $key . "\r\n";
        foreach ($headers as $h) $header.=$h."\r\n"; 
     ...
    }

```

**Comment by heminei** [comment]
The changes are released in version 1.7.0:

Client:

```
$client = $socket->connect("/", "example.com", [
	"Api-Key" => "111-222-333",
]);
```

Server: 
```
$socket->on("connect", function (\HemiFrame\Lib\WebSocket\Client $client) {
    var_dump($client->getHeader("Api-Key"));
});
```

**Comment by mdelclaro** [comment]
Thanks!!!!
